packages/govuk-frontend/src/govuk/init.mjs

```diff
@@ -90,7 +123,16 @@ function createAll(Component, config, $scope = document) {
           ? new Component($element, config)
           : new Component($element)
       } catch (error) {
-        console.log(error)
+        if (onError) {
+          onError(/** @type {Error} */ (error), {
```

Comment:
**suggestion** We've usually ran `typeof`/`instanceof` calls rather than casting to make typescript aware of types. Would the following work for that?

```suggestion
        if (onError && error instanceof Error) {
          onError(error, {
```